docs/ss_slide_003.md

```diff
@@ -2,19 +2,19 @@
 # Our first slide intuition makes sense for 'simple' physics
 
 {:style="text-align:center;"}
-![lenia title](https://raw.githubusercontent.com/riveSunder/yuca/master/assets/consequential_step_size/step_size_ball_drop.gif)
+![lenia title](https://raw.githubusercontent.com/riveSunder/yuca_docs/master/assets/consequential_step_size/step_size_ball_drop.gif)
 
 Ball drop in PyBullet [^pybullet] across 4 orders of magnitude of simulation step size: *a)* 1/24000 seconds per step, *b)* 1/2400 seconds per step, *c)* 1/240 seconds per step (PyBullet default), *d)* 1/24 seconds per step.
 {:style="text-align:center;"}
 
 {:style="text-align:center;"}
-![lenia title](https://raw.githubusercontent.com/riveSunder/yuca/master/assets/consequential_step_size/too_big_step_size_ball_drop.gif)
+![lenia title](https://raw.githubusercontent.com/riveSunder/yuca_docs/master/assets/consequential_step_size/too_big_step_size_ball_drop.gif)
 
 Ball drop in PyBullet [^pybullet] with a simulation step size of 1/2.4 seconds per step.
 {:style="text-align:center;"}
 
 {:style="text-align:center;"}
-[Previous slide](https://rivesunder.github.io/yuca/ss_slide_003a.md) -- [Next slide](https://rivesunder.github.io/yuca/ss_slide_004)
+[Previous slide](https://rivesunder.github.io/yuca_docs/ss_slide_003a.md) -- [Next slide](https://rivesunder.github.io/yuca_docs/ss_slide_004)
```

Comment:
Let's go ahead and revert the links to the gh-pages to point to `https://rivesunder.github.io/yuca` instead of yuca_docs. This should decrease navigation confusion while still offloading the larger assets storage to yuca_docs.